studio/app.yaml

```diff
@@ -1,7 +1,11 @@
-runtime: python27
```

Comment:
This probably needs to be revisited to reflect the policy on which python to use in gcloud.

Reply:
yeah, to be frank, I have not re-deployed the gae app; so I don't know if that part will work. But we don't have such policy just yet, if I understand correctly :) Using python3 may be helpful if we try to integrate the hosted UI with python notebooks in some way; but for now I am quite ambivalent on that.